src/routes/sessions/+layout.svelte

```diff
@@ -13,14 +14,20 @@
 	<svelte:fragment slot="list-items">
 		{#if $sessionsStore && $sessionsStore.length > 0}
 			<!-- Using slice() to reverse $sessionsStore without affecting the original array -->
-			{#each $sessionsStore.slice().reverse() as session (session.id)}
+			{#each $sessionsStore.slice().sort((a, b) => {
+				if (!a.updatedAt || !b.updatedAt) return 0;
+				return a.updatedAt < b.updatedAt ? 1 : -1;
+			}) as session}
 				{@const hasKnowledge = session.messages[0].knowledge}
 				{@const knowledgeName = session.messages[0].knowledge?.name}
 				<SectionListItem
 					sitemap={Sitemap.SESSIONS}
 					id={session.id}
 					title={hasKnowledge ? session.messages[1].content : session.messages[0].content}
-					subtitle={knowledgeName ? `${session.model} — ${knowledgeName}` : session.model}
+					subtitle={(knowledgeName ? `${session.model} — ${knowledgeName}` : session.model) +
+						(session.updatedAt
+							? ` - ${formatShortDistanceToNow(new Date(session.updatedAt), { addSuffix: true })}`
+							: '')}
```

Comment:
Hmm this is getting too complex, let's get rid of the string interpolated ` - `'s.

Instead I think we can do something like:
```svelte
{@const subtitles = [(knowledgeName ? session.model : knowledgeName), ... ]}

<SectoinListItem ... subtitle={subtitles.join(' - ')
```

Reply:
We also need to add this logic to [`Knowledge`](https://github.com/fmaclen/hollama/blob/main/src/routes/knowledge/%2Blayout.svelte#L16-L21)

Reply:
Great! I've already pushed this change (I've used `•` instead of `-` because I think it looks nicer, let me know what you think). Maybe in the future we could add these subtitles as badges, or leave the timestamp at the right of the title (in the same row) and use badges for the model and the knowledge.

Reply:
Gotcha, `•` does look a bit nicer 👍 

In the near future want to re-think how these details on the sidebar and on the headers work, I'm not 100% convinced that including all of that metadata in those places is actually useful. Knowledge, for example, is already included in the history of the session so it probably doesn't make sense to include it there.

But until then I just want things not to _look broken_ 😅